bundles/org.openhab.binding.haywardomnilogic/src/main/java/org/openhab/binding/haywardomnilogic/internal/handler/HaywardPumpHandler.java

```diff
@@ -213,11 +214,12 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                                     cmdString = pumpMaxSpeed;
                                 }
                             }
+                            break;
```

Comment:
Would you consider providing a separate PR with this bugfix? I can then cherry-pick it into 4.0.x so it will be included with the next hotfix. It seems very low risk. 😉

Reply:
Done!

Reply:
Btw, the `pumpSpeedRpm` channel:
https://github.com/openhab/openhab-addons/blob/67c0039cfdd46443707bc3e4dca61244b831a576/bundles/org.openhab.binding.haywardomnilogic/src/main/resources/OH-INF/thing/pump.xml#L48-L53

could now be `Number:Frequency` since rpm has been introduced as unit in 4.0. 🙂 See for example #15002.

Reply:
Oh, you are right!